seurat-find-transfer-anchor.R

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env Rscript
+
+suppressPackageStartupMessages(require(optparse))
+suppressPackageStartupMessages(require(workflowscriptscommon))
+
+#set parsing options
+
+option_list = list(
+ 
+  
+   make_option(
+    c("-i", "--query-file"),
+    action = "store",
+    default = NA,
+    type = 'character',
+    help = "seurat object to use as the reference",
+  ),
+ 
+   make_option(
+    c("--query-format"),
+    action = "store",
+    default = "seurat",
+    type = 'character',
+    help = "format of the query"
+  ),
+  make_option(
+    c("-r", "--reference-file"),
+    action = "store",
+    default = NA,
+    type = 'character',
+    help = "seurat object to use as the reference"
+  ),
+  
+  make_option(
+    c("--reference-format"),
+    action = "store",
+    default = "seurat",
+    type = 'character',
+    help = "format of the reference"
+  ),
+  
+  
+  make_option(
+    c("-o", "--output-file"),
+    action = "store",
+    default = NULL,
+    type = 'character',
+    help = "path to the output"
+  ),
+  make_option(
+    c("-n", "--normalization-method"),
+    action = "store",
+    default = 'SCT',
+    type = 'character',
+    help = "default SCT can also be LogNormalize"
```

Comment:
Please match help strings as closely as possible to the output of ?FindTransferAnchors. e.g. this one should be 'Name of normalization method used: LogNormalize or SCT' - though adding a '(default: <val>)' is fine.

Again: the default I see for this function is not SCT, it is 'LogNormalize'- please make sure default for all arguments match those of the R function.